tests/func/test_config.py

```diff
@@ -279,6 +279,16 @@ def test_load_relative_paths(dvc, field, remote_url):
     )
 
 
+def test_config_gdrive_fields(tmp_dir, dvc):
```

Comment:
Q: I don't see that many remote specific tests in this file (makes sense), but I would probably keep it here still, no need to complicate this at this point?